core/math.go

```diff
@@ -0,0 +1,14 @@
+package core
+
+import "math/big"
+
+func Pow(a *big.Float, e uint64) *big.Float {
+	if e == 0 {
+		return big.NewFloat(1)
+	}
+	result := big.NewFloat(0).Copy(a)
+	for i := uint64(0); i < e-1; i++ {
```

Comment:
if `e` contains a large number, this function will lead to large execution times. In what scenario is this used? Couldn't use the golang's built in pow function?

Reply:
reverted to the math.pow